sv_ttk/__init__.py

```diff
@@ -54,18 +54,15 @@ def get_theme(cls, root: tkinter.Misc | None = None) -> str:
     @classmethod
     def set_theme(cls, theme: str, root: tkinter.Misc | None = None) -> None:
         cls.load_theme(root)
-        if theme not in {"dark", "light"}:
+        if theme.lower() not in {"dark", "light"}: # Maybe someone will write Dark
             raise RuntimeError("not a valid sv_ttk theme name: {}".format(theme))
 
         cls.tcl.call("set_theme", theme)
```

Comment:
```suggestion
        cls.tcl.call("set_theme", theme.lower())
```

You will need to add `.lower()` here as well.

Reply:
LOL, akuli give the same change.